app/2.2.x/configuration.md

```diff
@@ -1393,7 +1398,7 @@ name   | description  | default
 **pg_password** | Postgres user's password. | none
 **pg_database** | The database name to connect to. | `kong`
 **pg_schema** | The database schema to use. If unspecified, Kong will respect the `search_path` value of your PostgreSQL instance. | none
-**pg_ssl** | Toggles client-server TLS connections between Kong and PostgreSQL. | `off`
+**pg_ssl** | Toggles client-server TLS connections between Kong and PostgreSQL. Since PostgreSQL uses the same port for TLS and non-TLS this is only a hint. If the server does not support TLS the established connection will be a plain one. | `off`
```

Comment:
```suggestion
**pg_ssl** | Toggles client-server TLS connections between Kong and PostgreSQL. Because PostgreSQL uses the same port for TLS and non-TLS, this is only a hint. If the server does not support TLS, the established connection will be a plain one. | `off`
```

Reply:
Thanks for the review. I've added these changes to the original config file and regenerated the docs. I will send other pull requests on the kong repo to make these changes permanent.